__tests__/api.test.js

```diff
@@ -275,3 +276,24 @@ describe("DELETE /api/comments/comment_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("200: responds with an array of objects with the properties of username, name and avatar_url.", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        const { body } = response; // body is the same as results from controller
+        expect(body).toHaveProperty("users");
+        expect(body.users).toHaveLength(4);
+        body.users.forEach((user) => {
+          expect(user).toHaveProperty("username", expect.any(String));
+          expect(user).toHaveProperty("name", expect.any(String));
+          expect(user).toHaveProperty("avatar_url", expect.any(String));
+        });
+      });
+  });
+  test("404: responds with an error message saying that the requested resource does not exist", () => {
```

Comment:
You only need one of these tests for a non-existent endpoint. Please make sure it is in its own describe block.